config/jobs/kubernetes/sig-k8s-infra/k8sio-presubmit.yaml

```diff
@@ -206,6 +207,13 @@ presubmits:
       - image: gcr.io/k8s-staging-infra-tools/k8s-infra:latest
         command:
         - ./hack/verify.sh
+        resources:
+          limits:
+            cpu: 1
+            memory: "512Mi"
```

Comment:
This seems really tiny but I'm willing to see what happens

Reply:
we still stay under 2m of execution: https://prow.k8s.io/?job=pull-k8sio-verify. 😅